checkov/common/output/secrets_record.py

```diff
@@ -93,6 +96,29 @@ def to_string(self, compact: bool = False, use_bc_ids: bool = False) -> str:
             splitted_record = processed_record.split("\n")
             splitted_record.insert(2, validation_status_message)
             processed_record = "\n".join(splitted_record)
+
+        processed_record = self._add_commit_details(processed_record)
+        return processed_record
+
+    def _add_commit_details(self, processed_record) -> str:
+        splitted_record = processed_record.split("\n")
+        file_idx = 0
+        file_line = ''
+        for idx, line in enumerate(splitted_record):
+            if line.__contains__('File:'):
+                file_idx = idx
+                file_line = line
+                break
+        added = False
+        if self.added_commit_hash:
+            file_line = file_line + f'; {COMMIT_ADDED_STR}: {self.added_commit_hash}'
+            added = True
+        if self.removed_commit_hash:
```

Comment:
I recommend to first check, if one of the hashes exist and then continue, otherwise just directly return the record